docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

```diff
@@ -0,0 +1,88 @@
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider using less, but more powerful, clusters
+* Keep into account that Kubernetes permissions are always "additive" (allow-list) rather than "subtractive" (deny-list). Whenever applicable, try to minimize configurations that gives access to "all but one aspect" (cluster, project, namespace...) as that will result in the creation of a high number of `RoleBindings`
+* Experiment to see if creating new projects or creating new clusters manifests in fewer `RoleBindings` for your specific use case.
+
+### RoleBinding count estimation
+
+Predicting exactly the number of `RoleBindings` a given configuration will create depends on many factors and is complicated to calculate. However, it is possible to give a first estimation according to considerations below:
+* As a minimum estimation consider the formula `32C + U + 2UaC + 8P + 5Pa`, where `C` is the cluster count, `U` is the user count, `Ua` is the average count of users with a membership on a cluster, `P` is the project count, and `Pa` is the average number of users with a membership on a project
+* The Restricted Admin role follows a different formula, as every user with Restricted Admin role will result in at least `7C + 2P + 2` additional `RoleBindings`
+* The number of `RoleBindings` generally increases linearly with cluster count, project count, and user count
+
+### Using New Apps Over Legacy Apps
+There are two app Kubernetes resources that Rancher uses: `apps.projects.cattle.io` and `apps.cattle.cattle.io`. Legacy apps, `apps.projects.cattle.io`, were first introduced with the former UI (Cluster Manager) and are now outdated. New apps, `apps.catalog.cattle.io`, are found in the current UI (Cluster Explorer) for their respective cluster. New apps are preferable because their data resides in downstream clusters, freeing up resources in the local cluster.
+
+It is recommended to remove any remaining legacy apps that appear in the Cluster Manager, replacing them with apps in the Cluster Explorer for their target cluster if necessary and creating any future apps in the cluster's Cluster Explorer only.
+
+### Using the Authorized Cluster Endpoint (ACE)
+An [Authorized Cluster Endpoint](../../../reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters#4-authorized-cluster-endpoint) option exist to access the Kubernetes API of Rancher provisioned RKE1, RKE2, and K3s clusters. When enabled this adds a context to generated kubeconfig files generated for the cluster that uses a direct endpoint to the cluster, thereby bypassing Rancher. That reduces load on Rancher for use cases where unmediated API access is acceptable or preferable.
+
+Note that, in order for `kubeconfig` to take advantage of ACE, users need to issue the `kubectl use-context <ace context name>` command in order to start using it.
+
+### Experimental: Option to Reduce Event Handler Executions
```

Comment:
It used to be, but it's not experimental any longer as several users have returned with positive reports (and the fleet project uses the equivalent by default since several versions).

In a future Rancher version, this option will also be made default (and this documentation will be updated accordingly).

Reply:
Maybe edit to "Optional: Reduce Event Handler Executions" so we avoid the "experimental" term entirely

Reply:
FMPOV it is not any more or less optional than the other steps in this guide - "Option" referred to the environment variable more than an optional step in the intention of the original text.

I changed it to just "Reducing Event Handler Executions" in 6ea7053b233 to follow the same pattern of previous paragraphs, let me know if that is not acceptable from your perspective.